engine/syncer.go

```diff
@@ -490,6 +490,19 @@ func (e *ExchangeCurrencyPairSyncer) Start() {
 		}
 
 		exchangeName := Bot.Exchanges[x].GetName()
+		if !Bot.Settings.EnableExchangeHTTPRateLimiter {
```

Comment:
This should be moved done on the actual exchange loading side in exchange.go in engine